js/bitstamp.js

```diff
@@ -99,6 +99,10 @@ module.exports = class bitstamp extends Exchange {
                         'sell/{pair}/',
                         'sell/market/{pair}/',
                         'sell/instant/{pair}/',
+                        'btc_withdrawal/',
+                        'btc_address/',
+                        'ripple_withdrawal/',
```

Comment:
What is the difference between ripple_withdrawal and xrp_withdrawal?

Reply:
XRP withdrawal is used for XRP only, while Ripple withdrawal is used for Ripple IOU (sending for example BTC or USD) on the Ripple network.

Reply:
@xsmedjax thank you!